Sources/PackageCollectionsModel/JSONPackageCollection.swift

```diff
@@ -1,17 +1,19 @@
 /*
  This source file is part of the Swift.org open source project
 
- Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Copyright (c) 2020-2021 Apple Inc. and the Swift project authors
  Licensed under Apache License v2.0 with Runtime Library Exception
 
  See http://swift.org/LICENSE.txt for license information
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
  */
 
+import Foundation
+
 public enum JSONPackageCollectionModel {}
```

Comment:
is the `JSON` prefix needed, since we are using Codable it could theoretically some different format? not a strong feeling, more of a question really

Reply:
amended https://github.com/apple/swift-package-manager/pull/3199/commits/da21ac1ae366b0bcead6342e8609aa88a1c2f48c